### PR TITLE
Port pAI remote control.

### PR DESCRIPTION
**Pull request by george99g**
Basically ports what remote control was before the rebase, with a few minor tweaks.
Most notably, disabling it for the robotics console.

Original code by @Alek2ander 

:cl:
rscadd: Ported AI Remote Control from Pre-Rebase.
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -39,6 +39,7 @@
 		/obj/item/stack/tile/fakespace/loaded					= 2,
 		/obj/item/toy/toy_xeno									= 2,
 		/obj/item/weapon/restraints/handcuffs/fake              = 2)
+	paiAllowed = 0
```
Comment (Cruix) [inline code/game/machinery/computer/arcade.dm]: So I can't order my pai to grind toys for me? What's even the point of remote control?

Comment (george99g) [inline code/game/machinery/computer/arcade.dm]: It was like that before the rebase, though.

**Comment by Alek2ander** [comment]
> disabling it for the robotics console

A rogue AI should depower it anyway.

Also, you should add the little light indicator so people can tell a computer is remote controlled

File: code/game/machinery/computer/arcade.dm (hunk 1 above)
Comment (Kmc2000) [inline code/game/machinery/computer/arcade.dm]: Let pais grind toys geroge

**Comment by george99g** [comment]
> Also, you should add the little light indicator so people can tell a computer is remote controlled

Let me go look for that particular commit...

**Comment by ParagonPrime** [comment]
IMO pAI should be able to control ALL the computers.

**Comment by george99g** [comment]
So the commit the indicator got added in was also a tgmerge.
Fuck me.

**Comment by Alek2ander** [comment]
kek

**Comment by george99g** [comment]
There. Everyone should be happy now.

**Comment by george99g** [comment]
untitled
Before anyone complains about a sprite PR without a screenshot.

**Comment by oisin100** [comment]
<3

**Pull request merged**